# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cf/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For packages utilizing imake ...

  - Rene Rebe <rene@exactcode.de>

diff -u cf/site.def cf-fixed/site.def
--- cf/site.def	2004-05-06 17:08:25.000000000 +0200
+++ cf-fixed/site.def	2005-11-04 14:36:26.000000000 +0100
@@ -72,7 +72,7 @@
 #ifdef AfterVendorCF
 
 #ifndef ProjectRoot
-#define ProjectRoot /usr/X11R6
+#define ProjectRoot /usr/X11
 #endif
 
 /*
